public/app/api/createBaseQuery.ts

```diff
@@ -0,0 +1,35 @@
+import { lastValueFrom } from 'rxjs';
+
+import { BackendSrvRequest, getBackendSrv, isFetchError } from '@grafana/runtime';
+
+interface RequestOptions extends BackendSrvRequest {
+  manageError?: (err: unknown) => { error: unknown };
+  showErrorAlert?: boolean;
+  body?: BackendSrvRequest['data'];
+}
+
+export function createBaseQuery({ baseURL }: { baseURL: string }) {
+  async function backendSrvBaseQuery(requestOptions: RequestOptions) {
+    try {
+      const { data: responseData, ...meta } = await lastValueFrom(
+        getBackendSrv().fetch({
+          ...requestOptions,
+          url: baseURL + requestOptions.url,
+          showErrorAlert: requestOptions.showErrorAlert,
```

Comment:
I vote for hard-coding (or defaulting) this to `showErrorAlert: false`, but not too precious about it.

Reply:
I think defaulting to `false` would be fine.